### PR TITLE
FluentBit: Collect kernel messages

### PR DESCRIPTION
**Pull request by mrIncompetent**
**What this PR does / why we need it**:
This adds the kmesg plugin to the fluentbit chart.

**Does this PR introduce a user-facing change?**:
```release-note
FluentBit can now collect the kernel messages
```

### CODE REVIEW

File: config/logging/fluentbit/templates/daemonset.yaml [hunk 1]
```diff
@@ -35,6 +35,9 @@ spec:
       - name: fluent-bit
         image: '{{ .Values.logging.fluentbit.image.repository }}:{{ .Values.logging.fluentbit.image.tag }}'
         imagePullPolicy: {{ .Values.logging.fluentbit.image.pullPolicy }}
+        securityContext:
+          # Required to read kmesg. TODO: Find decent capabilities
+          privileged: true
```
Comment (xrstf) [inline config/logging/fluentbit/templates/daemonset.yaml:41]: Please apply the same `if` around the security context as you did in the configmap template.

File: config/logging/fluentbit/templates/daemonset.yaml [hunk 2]
```diff
@@ -55,6 +58,9 @@ spec:
           readOnly: true
         - name: config
           mountPath: /fluent-bit/etc/
+        - name: kmsg
+          mountPath: /dev/kmsg
+          readOnly: true
```
Comment (xrstf) [inline config/logging/fluentbit/templates/daemonset.yaml:66]: Likewise, optional. Like the volume further down.

File: config/logging/fluentbit/templates/daemonset.yaml (hunk 1 above)
Comment (mrIncompetent) [reply config/logging/fluentbit/templates/daemonset.yaml:41]: Done with https://github.com/kubermatic/kubermatic/pull/4007/commits/9432e6ed80c692936f3e59301a53765402caf7e6

File: config/logging/fluentbit/templates/daemonset.yaml (hunk 2 above)
Comment (mrIncompetent) [reply config/logging/fluentbit/templates/daemonset.yaml:66]: Done with https://github.com/kubermatic/kubermatic/pull/4007/commits/3c1a4cf4dcda6e1886986719c0d7cdc6ddf745c5

**Pull request merged**